### PR TITLE
Featured items data

### PR DESCRIPTION
**Pull request by zoef-yt**
# Update on E-com page.
 In this pr, I have changed the featured items card from normal HTML repeating div to react component. 
Live Preview:- [Homepage](https://deploy-preview-2--need-games-zoef.netlify.app/)
### Updates
- [x] Added faker.js for now to show fake data.
- [x] Created a single "featured items card" which show different things based on the data passed
# Before

# After
### Images and data are shown based on the faker.js passed data 
image

### What not to check
- [ ] The functionality of the app
- [ ] Routing

### CODE REVIEW

File: src/components/homepage/homepage-body.jsx [hunk 1]
```diff
@@ -21,216 +22,52 @@ const HomepageBody = () => {
...
+					<p>{props.name}</p>
+					{props.inStock && <button className='btn btn-primary'>Buy Now</button>}
+					<p>Available: {props.type}</p>
+				</div>
+			</div>
+
+			{props.inStock ? (
+				props.hasOffer && (
+					<div className='card-badge'>
+						{' '}
```
Comment (dev-prabhat) [inline src/components/homepage/homepage-body.jsx]: remove these {" "} thing from code

File: src/data/featured-data.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const featuredData = [...Array(20)].map((item) => ({
+	id: faker.random.uuid(),
+	name: faker.commerce.productName(),
+	image: faker.random.image(),
+	inStock: faker.random.boolean(),
+	type: faker.random.arrayElement(['Consoles', 'Games', 'Accessories ']),
+	hasOffer: faker.random.boolean(),
+	offer: faker.random.arrayElement(['Save $20', '10% off', 'Buy one get one free', 'Stock clearance sale ']),
+
+	//! COMMENTED FOR FUTURE
```
Comment (dev-prabhat) [inline src/data/featured-data.jsx:12]: Everything looks fine remove commented code

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Good looks clean and non-repetitive only remove that commented code

**Review by humzahasan — COMMENTED** [review]
Do the changes @dev-prabhat suggested. Rest all looks good!

File: src/components/homepage/homepage-body.jsx (hunk 1 above)
Comment (zoef-yt) [reply src/components/homepage/homepage-body.jsx]: Done

File: src/data/featured-data.jsx (hunk 2 above)
Comment (zoef-yt) [reply src/data/featured-data.jsx:12]: That is for future reference

**Pull request merged**